src/frontend/src/pages/GanttPage/GanttPackage/README.md

```diff
@@ -0,0 +1,7 @@
+# Note
+
+We stole this from here: https://github.com/MaTeMaTuK/gantt-task-react
```

Comment:
lol what if we worded this slightly less sus

Reply:
commandeered